src/models.ts

```diff
@@ -20,9 +40,27 @@ export const baseMetrics: ReadonlyArray<BaseMetric> = [
   BaseMetric.AVAILABILITY
 ];
 
-export type BaseMetricValue = 'A' | 'C' | 'H' | 'L' | 'N' | 'P' | 'R' | 'U';
+export const temporalMetricMap: Metrics<TemporalMetric> = [
```

Comment:
as this is not a map but a list, i would like to have names like `baseMetrics`, `temporalMetrics`, `environmentalMetrics`

Reply:
It might be difficult to differentiate the two variables in the code with just a plural and case difference.
Maybe baseMetricList would do the trick ? What do you think ?

Reply:
I a revert those changes as you asked.